<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCustomer</class>
 <widget class="QDialog" name="AddCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open Customer Account</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="pushButton_back">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0);
font: 11pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>450</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>25</width>
         <height>450</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>50</y>
          <width>388</width>
          <height>311</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Name</string>
                 </property>
                 <property name="buddy">
                  <cstring>lineEdit_name</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_name"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Father's Name</string>
                 </property>
                 <property name="buddy">
                  <cstring>lineEdit_father</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_father"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Mother's Name</string>
                 </property>
                 <property name="buddy">
                  <cstring>lineEdit_mother</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_mother"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Mobile Number</string>
                 </property>
                 <property name="buddy">
                  <cstring>lineEdit_mob</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_mob"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Email</string>
                 </property>
                 <property name="buddy">
                  <cstring>lineEdit_email</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_email"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>ID Number</string>
                 </property>
                 <property name="buddy">
                  <cstring>lineEdit_id</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_id"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>17</width>
               <height>28</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_submit">
               <property name="styleSheet">
                <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0);
font: 11pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Submit</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_reset">
               <property name="styleSheet">
                <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0);
font: 11pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_father</tabstop>
  <tabstop>lineEdit_mother</tabstop>
  <tabstop>lineEdit_mob</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>pushButton_submit</tabstop>
  <tabstop>pushButton_reset</tabstop>
  <tabstop>pushButton_back</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_mother</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_father</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_mob</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_email</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_id</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
